draw_app/app.py

```diff
@@ -13,12 +15,17 @@ def index():
 
 @app.route('/draw/')
 def draw_html():
-    return render_template("draw.html", items=global_drawer.draw())
+    return render_template("draw.html", items=json.loads(draw()))
 
 
 @app.route('/draw/api/daily')
 def draw():
     new_draw = global_drawer.draw()
+    points_container = PointsContainer()
+
+    for p in new_draw.keys():
+        new_draw[p] = {"points": points_container.get_points(p), "quote": new_draw[p]}
```

Comment:
`new_draw = {p: {"points": points_container.get_points(p), "quote": q} for p, q in new_draw.items()}`

Reply:
Also this seems to shoot a query to the DB for each player separately, instead of getting all scores at once and merging them with the quotes (we have the quotes in memory anyway)